hazelcast/src/test/java/com/hazelcast/client/protocol/compatibility/ErrorCodecCompatibilityTest.java

```diff
@@ -38,22 +40,23 @@
 
 @RunWith(Parameterized.class)
 @Parameterized.UseParametersRunnerFactory(HazelcastParametersRunnerFactory.class)
+@Category(NightlyTest.class)
```

Comment:
If this test is important for code coverage, then it has to be a `SlowTest` or `QuickTest`. If it's just for correctness, then `NightlyTest` is absolutely fine.